tutorial/practicals/nemo/1_nemo_psyir/README.md

```diff
@@ -173,7 +195,7 @@ interesting from a performance point of view.
 To familiarise yourself with PSyIR navigation, you can:
 
 1. Modify the transformation script so that it breaks-out into the Python
-   debugger once it has obtained the PSyIR:
+   debugger, pdb, as soon as it is called (from PSyclone):
 
```

Comment:
At line 184, is "it be fully understood" proper english?

Reply:
Also, in this paragraph I wouldn't go into fparser details. Instead of "For more information on fparser ..." I would replace it with "PSyclone treats the CodeBlock as a blackbox of code that cannot transform, but still enables the transformation of the remaining code outside the CodeBlock."

Reply:
To answer your first question - yes it's fine. I've updated the paragraph as you suggest.